README.md

```diff
@@ -0,0 +1,120 @@
+# TechJobs, Console Edition Autograded Rubric
+
+For [this assignment](https://education.launchcode.org/csharp-web-development/assignments/tech-jobs-console.html), students are asked to complete a console app that allows users to browse, search, and print listings of open jobs.
+
+## Assignment Requirements
+
+1. Implement the 'printJobs' method
+2. Create the `findByValue` method
+3. Make search methods `Case-Insensitive`
+
+## Unit Tests 
+
+The starter code contains unit tests and text files. There is one test class that contains 4 tests. These tests compare students' output to the text files.  
+
+We encourage students to run unit tests after they complete the assignment before submitting. Directions for how to run tests have been provided in the [Java](https://education.launchcode.org/java-web-development/) textbook's [Assignment 0](https://education.launchcode.org/csharp-web-development/assignments/hello-world.html).
```

Comment:
The first link is going to the Java Book not C#

Reply:
Fixed!